client_cli/src/main.rs

```diff
@@ -63,8 +63,8 @@ impl FromStr for Configuration {
     fn from_str(file: &str) -> Result<Self> {
         let deser_err_msg = format!("Failed to decode config file {} ", &file);
         let err_msg = format!("Failed to open config file {}", &file);
-        let file = File::open(file).wrap_err(err_msg)?;
-        let cfg = serde_json::from_reader(file).wrap_err(deser_err_msg)?;
+        let content = fs::read_to_string(file).wrap_err(err_msg)?;
```

Comment:
Buffering this way slightly increases RAM usage. But usually faster, @ilchu keep this in mind in the future.

Reply:
That's how I've done it in the config's `LoadFromDisk` trait. As these are usually called once on startup, hope the effect is almost negligible.

Reply:
I like how it was done before, but unfortunately `json5` crate has nothing like `serde_json::from_reader()`